frontend/src/utils/ropeApi.ts

```diff
@@ -0,0 +1,65 @@
+export interface User {
+  email: string
+  isAdmin: boolean
+  isManager: boolean
+  id: number
+}
+
+export const ropeApi = {
+  fetchUsers: async (): Promise<User[]> => {
+    const response = await fetch('/api/user')
+    if (!response.ok) {
+      throw new Error('Failed to get users')
+    }
+    const usersFromApi = await response.json()
+    const users: User[] = usersFromApi.map((user: { is_admin: boolean, is_manager: boolean }) => ({
+      ...user,
+      isAdmin: user.is_admin,
+      isManager: user.is_manager
+    }))
+    return users
```

Comment:
For our future selves, we may want to do the same in the return objects for `addUser` and `updateUserPermissions`. Maybe this can be lifted to a helper function which all three use to perform the field mapping.

Reply:
:thinking: -- Actually, right now if you add a user and then try to update fields, you may hit the same `422` (?), so maybe this is also required for our current selves :joy: .